CDP4Common.Tests/Poco/OptionTestFixture.cs

```diff
@@ -46,5 +49,31 @@ public void VerifyThatIsDefaultWorks()
             Assert.IsTrue(option1.IsDefault);
             Assert.IsFalse(option2.IsDefault);
         }
+
+        [Test]
+        public void VerifyGetNestedParameterValuesByPath()
```

Comment:
This seems like a duplicate of the test in `CDP4Common.NetCore.Tests/Poco/OptionTestFixture.cs`.
Do we need to have it in both places?

Reply:
Guess this question applies to the `CDP4Common.Tests/Helpers/NestedElementTreeGeneratorTestFixture.cs` as it seems to be identical (at a glance) with `CDP4Common.NetCore.Tests/Helpers/NestedElementTreeGeneratorTestFixture.cs`.

Is there any way to remove this duplication?

Reply:
Good question. It has been this way since I joined the company. Don't know if it is still needed.

Reply:
we have a dedicated testsuite for the .NET framework and the netstandard code. If you can propose a way to not have to duplicate i am all ears.